x-pack/package.json

```diff
@@ -225,6 +262,7 @@
     "react-datetime": "^2.14.0",
     "react-dom": "^16.3.0",
     "react-dropzone": "^4.2.9",
+    "react-markdown": "^3.4.1",
```

Comment:
Please upgrade Kibana to version 3.4.1 of react-markdown too, assuming it's just a minor upgrade.

Reply:
We currently only check for version matches amongst devDependencies